fe/src/components/Tab/ClientTab.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <v-app  height="30px" width="5px">
```

Comment:
you may remove this and change it into container since I would duplicate in the app.vue file